src/Nethermind/Nethermind.State/IWorldState.cs

```diff
@@ -86,7 +86,7 @@ public interface IWorldState : IJournal<Snapshot>, IReadOnlyStateProvider
     void CreateAccount(Address address, in UInt256 balance, in UInt256 nonce = default);
     void CreateAccountIfNotExists(Address address, in UInt256 balance, in UInt256 nonce = default);
 
-    void InsertCode(Address address, in ValueHash256 codeHash, ReadOnlyMemory<byte> code, IReleaseSpec spec, bool isGenesis = false);
+    bool InsertCode(Address address, in ValueHash256 codeHash, ReadOnlyMemory<byte> code, IReleaseSpec spec, bool isGenesis = false);
```

Comment:
maybe we should add a comment here to specify what this return bool means.
From first look at the interface someone might conclude that
true -> new code was set for the account
false -> the code for the account was same as before, so not updated

Reply:
Done